src/libman/Commands/CacheCommand.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.Web.LibraryManager.Tools.Commands
+{
+    /// <summary>
+    /// Defines the cache command for libman.
+    /// Child commands: clean, list
+    /// </summary>
+    internal class CacheCommand : BaseCommand
+    {
+        public CacheCommand(IHostEnvironment hostEnvironment, bool throwOnUnexpectedArg = true) 
+            : base(throwOnUnexpectedArg, "cache", Resources.CacheCommandDesc, hostEnvironment)
+        {
+        }
+
+        public override BaseCommand Configure(CommandLineApplication parent)
+        {
+            base.Configure(parent);
+
+            Commands.Add(new CacheCleanCommand(HostEnvironment).Configure(this));
+            Commands.Add(new CacheListCommand(HostEnvironment).Configure(this));
```

Comment:
We talked about disabling this command. I don't think we should list the cache contents.